conduit/plugins/processors/filterprocessor/expression/expression.go

```diff
@@ -56,10 +56,20 @@ func (e *regexExpression) Search(input interface{}) (bool, error) {
 	}
 }
 
-func makeRegexExpression(searchStr string, expressionType FilterType) (Expression, error) {
-	if expressionType != EqualToFilter && expressionType != RegexFilter {
-		return nil, fmt.Errorf("target type (string) does not support %s filters", expressionType)
+type stringEqualExpression struct {
+	Str string
+}
+
+func (e *stringEqualExpression) Match(input interface{}) (bool, error) {
+	switch v := input.(type) {
+	case string:
+		return e.Str == v, nil
+	default:
+		return false, fmt.Errorf("unexpected regex search input type (%T)", v)
 	}
+}
```

Comment:
Previously this was using a special regex expression. In benchmarks using string equals was 8-12% faster.